ang/civiawards/award-creation/directives/award.directive.html

```diff
@@ -14,7 +14,7 @@ <h1 crm-page-title>{{pageTitle}}</h1>
         Continue
       </button>
       <button
-        ng-click="navigateToDashboard()" class="btn btn-secondary-outline">
+        ng-click="navigateToPreviousPage()" class="btn btn-secondary-outline">
```

Comment:
Why did we have to change this?

Reply:
Ah, this was an error that I captured when I was working on this ticket. It is unrelated, but I took the occasion for fixing it.

Reply:
In that case can you mention the issue on the PR details then? Thanks

Reply:
I added a mention of the issue on the PR description.